lib/pure/mersenne.nim

```diff
@@ -53,3 +56,29 @@ proc getNum*(m: var MersenneTwister): uint32 =
   result = result xor ((result shl 7'u32) and 0x9d2c5680'u32)
   result = result xor ((result shl 15'u32) and 0xefc60000'u32)
   result = result xor (result shr 18'u32)
+
+proc getSeq*(mt: var MersenneTwister, len: int): seq[int] {.deprecated:
+"deprecated, use random.rand() instead"} =
+  ## Returns seq of pseudorandom ints len long.
+  for i in 1..len:
+    result.add(int(getNum(mt)))
+
+proc sample*[T](mt: var MersenneTwister, arr: seq[T]): T {.deprecated:
+"deprecated, use random.sample() instead"} =
+  ## Takes random sample of an seq[T].
+  var correspondingValues: seq[uint32]
+  var maxVal: uint32 = uint32(4294967295)
+  let maxValDivLenArr: uint32 = uint32(float32(maxVal) / float32(len(arr)))
+  for i in arr:
+    correspondingValues.add(maxVal)
+    maxVal = maxVal - maxValDivLenArr
+  let randNum = uint32(getNum(mt))
+  var largerNumIndex: uint32 = 0
+  var smallerNumIndex: uint32 = 1
+  while largerNumIndex < uint32(len(correspondingValues))  and smallerNumIndex < uint32(len(correspondingValues)):
+   if randNum < correspondingValues[largerNumIndex] and randNum > correspondingValues[smallerNumIndex]:
+    return arr[largerNumIndex]
+   elif randNum < correspondingValues[correspondingValues.high]:
+     return arr[correspondingValues.high]
+   largerNumIndex = largerNumIndex + 1
+   smallerNumIndex = smallerNumIndex + 1
```

Comment:
If the module is going to be deprecated anyway, what's the point in adding these functions?

Reply:
I will remove that, sorry.